### ISSUE TITLE
check-clang-checker warnings

### ISSUE THREAD

**Issue by madrisan** [madrisan/nagios-plugins-linux#24]
**Describe the bug**
The *clang* static analyser v8 (`/usr/bin/clang-8`) reports the following issue:

    tslibcontainer_count.c:83:7: warning: 1st function call argument is an uninitialized value
          free (perfdata);
          ^~~~~~~~~~~~~~~

**To Reproduce**
Steps to reproduce the behaviour:

    make -C tests check-clang-checker

**Expected behaviour**
This warning should not appear in the output of the *clang* static analyser.

**Additional context**
 - Fedora Linux 30 Workstation
 - Nagios Plugins Linux version 24

**Comment by madrisan** [comment]
Should be a false positive. Anyway this patch will probably solve the issue:
```
--- ./tests/tslibcontainer_count.c      2018-08-02 22:42:30.270807160 +0200
+++ ./tests/tslibcontainer_count.c.fix  2019-06-20 20:26:59.915199662 +0200
@@ -74,7 +74,7 @@
 {
   const struct test_data *data = tdata;
   int err, ret = 0;
-  char * perfdata;
+  char * perfdata = NULL;
   unsigned int containers;
 
   err = docker_running_containers (&containers, data->image, &perfdata, false);
```

**Comment by madrisan** [comment]
Fixed by the [Pull Request #31(https://github.com/madrisan/nagios-plugins-linux/pull/31).
